packages/kernel-6.1/kernel-6.1.spec

```diff
@@ -1312,6 +1312,7 @@ install -p -m 0644 %{S:302} %{buildroot}%{_cross_bootconfigdir}/05-metal.conf
 %{_cross_kmoddir}/kernel/net/netfilter/xt_TPROXY.ko.*
 %{_cross_kmoddir}/kernel/net/netfilter/xt_TRACE.ko.*
 %{_cross_kmoddir}/kernel/net/netfilter/xt_u32.ko.*
+%{_cross_kmoddir}/kernel/net/netlink/netlink_diag.ko.*
```

Comment:
Why is this the only spec file change? All three kernel configs look as if they should create a kmod.

Reply:
The `modules` section in the other two kernel spec files are not as explicit. We are choosing exactly which modules we want in 6.1 but 5.10 and 5.15 basically include all the modules via this: https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/develop/packages/kernel-5.10/kernel-5.10.spec#L302. So the specific break out is required only for 6.1. This is why I tried the other kernels explicitly to make sure the module is loadable on a booted box.